src/jni_core/src/jvm_runtime.rs

```diff
@@ -204,11 +204,32 @@ pub fn execute_with_jni_env<T>(
     jvm: &JavaVM,
     f: impl FnOnce(&mut JNIEnv<'_>) -> anyhow::Result<T>,
 ) -> anyhow::Result<T> {
-    let _guard = jvm
+    let mut env = jvm
         .attach_current_thread()
         .with_context(|| "Failed to attach current rust thread to jvm")?;
 
-    let mut env = jvm.get_env().with_context(|| "Failed to get jni env")?;
+    // set context class loader for the thread
+    // java.lang.Thread.currentThread()
+    //     .setContextClassLoader(java.lang.ClassLoader.getSystemClassLoader());
```

Comment:
dead code?

Reply:
This is the equivalent java code, I think it's for reference purpose.